indexers/Caddyfile

```diff
@@ -7,6 +7,14 @@
     }
 }
 
+# Define basic auth
+(auth) {
+    basicauth {
+        # You can generate BCrypt hash using: caddy hash-password
+        admin $2a$14$A4H4GYb/HkbXKJXaib4.HObVwlGt0W.8WUihFg5U2KpTa8eODDvqK
```

Comment:
is this the password, if so, should it be commited?

Reply:
@marc-aurele-besner it's the hash of the password.